x-pack/plugins/apm/readme.md

```diff
@@ -141,7 +141,7 @@ node scripts/eslint.js x-pack/legacy/plugins/apm
 APM behaves differently depending on which the role and permissions a logged in user has. To create the users run:
 
 ```sh
-node x-pack/plugins/apm/scripts/create-apm-users-and-roles.js --username elastic --password changeme --kibana-url http://localhost:5601 --role-suffix <github-username-or-something-unique>
+node x-pack/plugins/apm/scripts/create-apm-users-and-roles.js --username admin --password changeme --kibana-url http://localhost:5601 --role-suffix <github-username-or-something-unique>
```

Comment:
When using apm-integration-testing the superuser with access to kibana is `admin`